pt-BR/book/reedline_nu_editor_de_linha.md

```diff
@@ -0,0 +1,675 @@
+
+A estrutura necessária para o registro é a seguinte:
+
+```nu
+{
+  value:       # The value that will be inserted in the buffer
+  description: # Optional. Description that will be display with the selected value
+  span: {      # Optional. Span indicating what section of the string will be replaced by the value
+    start:
+    end:
+  }
+  extra: [string] # Optional. A list of strings that will be displayed with the selected value. Only works with a description menu
+}
+```
+Para que o menu exiba algo, pelo menos o campo value deve estar presente no registro resultante.
+
+Para tornar o menu interativo, duas variáveis estão disponíveis no bloco: `$buffer` e `$position`. O `$buffer` contém o valor capturado pelo menu; quando a opção `only_buffer_difference` é verdadeira, `$buffer` é o texto escrito após a ativação do menu. Se `only_buffer_difference` for falso, `$buffer` é toda a string na linha. A variável `$position` pode ser usada para criar spans de substituição com base na ideia que você tinha para o seu menu. O valor de `$position` muda com base em se `only_buffer_difference` é verdadeiro ou falso. Quando é verdadeiro, `$position` é a posição inicial na string onde o texto foi inserido após a ativação do menu. Quando o valor é falso, `$position` indica a posição real do cursor.
+
+Usando essas informações, você pode projetar seu menu para apresentar as informações que você precisa e substituir esse valor na localização desejada. A única coisa extra que você precisa para interagir com seu menu é definir uma tecla de atalho que ative seu menu recém-criado.
+
+## Teclas de Atalho para Menus
+
+Caso deseje alterar a maneira padrão como ambos os menus são ativados, você pode fazer isso definindo novas teclas de atalho. Por exemplo, as duas teclas de atalho a seguir atribuem os menus de conclusão e de histórico para Ctrl+t e Ctrl+y, respectivamente:
+
+```nu
+$env.config = {
+    ...
+
+    keybindings: [
+      {
+        name: completion_menu
+        modifier: control
+        keycode: char_t
+        mode: [vi_insert vi_normal]
+        event: {
+          until: [
+            { send: menu name: completion_menu }
+            { send: menupagenext }
+          ]
+        }
+      }
+      {
+        name: history_menu
+        modifier: control
+        keycode: char_y
+        mode: [vi_insert vi_normal]
+        event: {
+          until: [
+            { send: menu name: history_menu }
+            { send: menupagenext }
+          ]
+        }
+      }
+    ]
+
+    ...
+  }
+```
+**Edit this page on <a href=https://github.com/nushell/nushell.github.io/edit/main/book/line_editor.md>GitHub**</a>
```

Comment:
This is the line that broken the CI, could be changed to `**Edit this page on [GitHub](https://github.com/nushell/nushell.github.io/edit/main/book/line_editor.md)**`